integration_tests/src/main/python/date_time_test.py

```diff
@@ -14,18 +14,28 @@
 
 import pytest
 from asserts import assert_gpu_and_cpu_are_equal_collect, assert_gpu_fallback_collect, assert_gpu_and_cpu_error, assert_gpu_and_cpu_are_equal_sql
-from conftest import is_utc, is_supported_time_zone, get_test_tz
+from conftest import is_utc, get_test_tz
 from data_gen import *
 from datetime import date, datetime, timezone
 from marks import allow_non_gpu, approximate_float, datagen_overrides, disable_ansi_mode, ignore_order, incompat, tz_sensitive_test
 from pyspark.sql.types import *
 from spark_session import with_cpu_session, is_before_spark_330, is_before_spark_350
 import pyspark.sql.functions as f
+from zoneinfo import ZoneInfo
+import pytz
 
 # Some operations only work in UTC specifically
 non_utc_tz_allow = ['ProjectExec'] if not is_utc() else []
 # Others work in all supported time zones
-non_supported_tz_allow = ['ProjectExec'] if not is_supported_time_zone() else []
+last_supported_tz_time = datetime(2200, 12, 30, 23, 59, 59, 999999, tzinfo=timezone.utc)
```

Comment:
Can we update this to indicate that it is the last time that is configured to be supported by the GPU transition rules.

A comment is good enough.